pkgs/by-name/ko/kotlin-lsp/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  openjdk,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kotlin-lsp";
+  version = "0.253.10629";
```

A [new version has been released](https://github.com/Kotlin/kotlin-lsp/releases/tag/kotlin-lsp/v261.13587.0).

```suggestion
  version = "261.13587.0";
```